app/controllers/api/v1/employees_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # Employees API controller
+    class EmployeesController < ApplicationController
+      before_action :authenticate_user!
+      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+      rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
+      rescue_from ActionPolicy::Unauthorized, with: :handle_unauthorized
+      rescue_from ArgumentError, with: :params_invalid
+
+      def index
+        authorize! Object, with: Employees::UserPolicy
+        @members = company.members
+        render json: @members, each_serializer: UserSerializer
+      end
+
+      def show
+        authorize! member, with: Employees::UserPolicy
+        render json: member, serializer: UserSerializer
+      end
+
+      def create
+        authorize! Object, with: Employees::UserPolicy
+        Employees::MemberInviter.call(member_params, company)
```

Comment:
I still don't like this moment, we are creating user in invite, and after that looking for the same user in db.
How about Service for creation (which returns user) + hook in service for notification?